eng/pipelines/release-go-images-pipeline.yml

```diff
@@ -146,6 +150,11 @@ extends:
                       -azdopat '$(System.AccessToken)'
                   displayName: ⌚ Wait for go-images build
                   timeoutInMinutes: 120
+
+                - script: |
+                    releasego generate-announcement -o $(Pipeline.Workspace)/AnnouncementData
```

Comment:
I might be missing something but where is the `generate-announcement` code? Is is this coming in a follow-up PR?